embulk-input-jdbc/src/main/java/org/embulk/input/JdbcInputPlugin.java

```diff
@@ -81,8 +80,10 @@ protected JdbcInputConnection newConnection(PluginTask task) throws SQLException
             // TODO check Class.forName(driverClass) is a Driver before newInstance
             //      for security
             driver = (Driver) Class.forName(t.getDriverClass()).newInstance();
-        } catch (Exception ex) {
-            throw Throwables.propagate(ex);
+        } catch (final RuntimeException | Error ex) {
```

Comment:
is `catch Error` needed because we just re-throw it again?

Reply:
Ah, right. `Error` is not `Exception` while `RuntimeException` is `Exception`. Good catch. Will address this.

Reply:
Fixed at https://github.com/embulk/embulk-input-jdbc/pull/207/commits/bc8ebcb911f6dd3d817521b04142548f5fe5d228#diff-978621b46b8e1eb4b8eb4e8cea26c09e229e1bc405e1f8d998b1f69b481c413dR83